### PR TITLE
Add initial Sept newsletter

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

File: newsletter-september-2022.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+-----------------
+
+- We continue to work on improvements to our URL handling code, which will allow us to support more flexible URL configurations for projects.
+- We are working to redesign some of our search to make it nicer across our dashboard and in-doc search experiences. 
+- Our telemetry data is starting to really come together, and we're gathering insights on interesting metrics around builds. Expect a few blog posts about this in the coming months.
+- The 1.1 release of our `sphinx_rtd_theme` had a large amount of work done, and is nearing release.
+
+Possible issues
+---------------
+
```
Comment (stsewd) [inline newsletter-september-2022.rst:46]: Maybe list https://github.com/readthedocs/readthedocs.org/issues/9548, but probably also makes sense to have a its own entry... maybe email users even (after we have checked that the fix actually works of course :D).

Comment (ericholscher) [reply newsletter-september-2022.rst:46]: Ah yea, good point.

File: newsletter-september-2022.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+and we need to continue making them easier to use.
+
+New features
+------------
+
+The latest updates from our team:
+
+- 🎉 We have soft launched the `initial version of our new landing pages <https://about.readthedocs.com/>`_ and started sharing it with the community. We plan to eventually have this as our primary landing page once we've gotten more feedback on it. 
+- We have improved out logic for `cancelling builds when a commit is pushed to an already building version <https://github.com/readthedocs/readthedocs.org/pull/9549>`_. This will improve user experience waiting on a build that will be overwritten, and save our servers some time as well!
+- We continued improving our docs pages. This month we did `badges <https://github.com/readthedocs/readthedocs.org/pull/9580>` & `build troubleshooting <https://docs.readthedocs.io/en/stable/build-troubleshooting.html>`_.
```
Comment (humitos) [inline newsletter-september-2022.rst:30]:
```suggestion
- We continued improving our docs pages. This month we did `badges <https://docs.readthedocs.io/en/stable/badges.html>` & `build troubleshooting <https://docs.readthedocs.io/en/stable/build-troubleshooting.html>`_.
```

File: newsletter-september-2022.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
+
+Upcoming features
+-----------------
+
+- We continue to work on improvements to our URL handling code, which will allow us to support more flexible URL configurations for projects.
+- We are working to redesign some of our search to make it nicer across our dashboard and in-doc search experiences. 
+- Our telemetry data is starting to really come together, and we're gathering insights on interesting metrics around builds. Expect a few blog posts about this in the coming months.
+- The 1.1 release of our `sphinx_rtd_theme` had a large amount of work done, and is nearing release.
```
Comment (humitos) [inline newsletter-september-2022.rst:42]:
```suggestion
- The 1.1 release of our ``sphinx_rtd_theme`` had a large amount of work done, and is nearing release.
```

File: newsletter-september-2022.rst [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+but we do plan to be more active in removing these features in the coming months.
+
+We continue to actively deprecate jQuery from our code, as well as `guide the Sphinx ecosystem <https://github.com/sphinx-doc/sphinx/issues/10608>`_ through the transition. 
+
+----
+
+Considering using Read the Docs for your next documentation project?
+Check out `our documentation <https://docs.readthedocs.io/>`_ to get started!
+
+Questions? Comments? Ideas for the next newsletter? `Contact us`_!
```
Comment (humitos) [inline newsletter-september-2022.rst:64]: I was thinking if it does make sense to "promote a feature" in this closing paragraph on each newsletter. Something small that fits in a tip paragraph. For example:

```rst
.. tip::

  Did you know that you can connect your project to Slack? Get notified instantly, 
  and don't miss a failing build again! Full documentation at https://docs.readthedocs.io/en/stable/build-notifications.html
```

What do you think?

**Review by humitos — APPROVED** [review]
Looks good to me!

File: newsletter-september-2022.rst (hunk 4 above)
Comment (ericholscher) [reply newsletter-september-2022.rst:64]: @humitos I think that's a great idea!

Comment (humitos) [reply newsletter-september-2022.rst:64]: Yes, that would work better! Actually, that is exactly what I wanted originally, but I didn't know how to suggest doing it 😅 Happy that you mentioned it!

Comment (ericholscher) [reply newsletter-september-2022.rst:73]: Done!

**Pull request merged**